src/framework/mlt_frame.c

```diff
@@ -470,7 +472,7 @@ int mlt_image_format_size( mlt_image_format format, int width, int height, int *
 			return 4;
 		case mlt_image_yuv422p16:
 			if ( bpp ) *bpp = 0;
-			return 4 * height * width ;
+			return height * ALIGN_TO( width * 2, 32 ) + 2 * height * ALIGN_TO( width, 32 ) ;
```

Comment:
How about if we declare that yuv422p16 shall be aligned and all other (previous) formats are not? I know it would not be consistent, but it might be a way to transition to a new method going forward.

Also, in the future, someone could add a format like rgb24_aligned, so that the exiting formats could eventually be aligned as well.